### ISSUE TITLE
Refactor landing page and login page to reduce unneccessary clicking/navigation

### ISSUE THREAD

**Issue by pacharanero** [rcpch/rcpch-audit-engine#756]
Several observations about the E12 landing page:

* most users of E12 will nbot really need to see info about what E12 is
* icons/images are HUGE and take up too much space
* the only important section: "By accessing, browsing, entering data or otherwise accessing the Site, you become a ‘user’ and assert that you have read and understand and agree to be bound by the terms of Agreement. If you have any questions about this Disclaimer, please contact us." could be higher up so as to be more obvious, perhaps highlighted in red or something

Also:

* we would like it so that when you go to the web root of the project you should immediately be presented with the login form. There's no reason not to simply put the login form right on the main page
* public facing stuff could be prominently linked to from that same login page for those who can't login
* Landing/login page shouldn't need to scroll - it should be a simgle page, footer visible, no scrolling

All in all there's a good refactor here to be done, which would simplify and streamline the login process

**Comment by anchit-chandran** [comment]
Agreed the landing page design could do with an overall. We've been through a first-pass iteration with @AmaniKrayemRCPCH just in terms of colors / texts etc but a UX design overhaul would be good
